relay/cmd/config.go

```diff
@@ -106,3 +106,10 @@ func NewConfig(ctx *cli.Context) (Config, error) {
 	}
 	return config, nil
 }
+
+// SanitizedConfig returns a copy of the Config with sensitive information removed or obfuscated.
+func (c Config) SanitizedConfig() Config {
+	sanitized := c
+	sanitized.RelayConfig.AuthenticationKeyCacheSize = 0 // Obfuscate sensitive information
```

Comment:
hmm this doesn't seem like a secret. It's just a cache size right? Maybe codeQL is just parsing "AuthenticationKey" and is not smart enough to understand the suffix?

Reply:
there is aws `SecretAccessKey` but it isn't triggering anything for codeQL. None other config seems to be a secret. maybe we just add shutup?

Reply:
Let’s not log secretaccesskey and shut up the rest.
Still, for best practices going forward, I’m now a big fan of separating configs and secrets into two separate structs, and only logging configs on startup. This I feel is a safer way to not in the future add more secrets that would be logged and not picked up by the now shut up codeQL.